force-app/main/default/classes/SOQL.cls

```diff
@@ -50,33 +61,58 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> getContactIdsByLastName() {
+        List<Contact> contacts = [
+            SELECT Id 
+            FROM Contact 
+            WHERE LastName 
+            IN ('Smith', 'Jones')
+            ];
+
+        List<Id> contactIds = new List<Id>();
+        for (Contact con : contacts) {
+            contactIds.add(con.Id);
+        }
```

Comment:
There is a special constructor for Map<Id, SObject> that takes a List<SObject> and separates it into a map of the Id to the record itself. You can use this to return the keySet() of a map, which is a bit simpler (and somehow faster) than doing the iteration on lines 72 through 75.